solver/overlays/test/kustomization.yaml

```diff
@@ -1,8 +1,10 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - ../../base
   - thoth-notification.yaml
+  - openshift-templates/solve_and_sync-template.yaml
+  - argo-workflows/solver-template.yaml
+  - argo-workflows/parse-solver-output.yaml
```

Comment:
Please move them under: `patchesStrategicMerge:`

Reply:
Also please copy over the openshift-template to test, it has to be tested
please don't move it . https://github.com/thoth-station/thoth-application/pull/482/files#diff-6da397dde79165a7081b95c1cd493ba6